docs-en/chapter_preface/about_the_book.md

```diff
@@ -1,45 +1,45 @@
+- **Complexity Analysis**: aspects and methods for evaluating data structures and algorithms. Methods of deriving time complexity and space complexity, with the common types, examples, etc.
+- **Data Structures**: basic data types, classification methods of data structures. Definition, pros and cons, common operations, common types, typical applications, implementation methods of data structures such as array, linked list, stack, queue, hash table, tree, heap, graph, etc.
+- **Algorithms**: definitions, pros and cons, efficiency, application scenarios, steps for solving questions, sample questions for algorithms such as search, sorting, divide and conquer, backtracking, dynamic programming, greedy algorithms, and more.
 
 ![Hello Algo content structure](about_the_book.assets/hello_algo_mindmap.jpg)
 
 ## Acknowledgements
 
 During the creation of this book, I received help from many people, including but not limited to:
 
-- Thank you to my mentor at the company, Dr. Shih Lee, for encouraging me to "get moving" during one of our conversations, which strengthened my resolve to write this book.
-- I would like to thank my girlfriend Bubbles for being the first reader of this book, and for making many valuable suggestions from the perspective of an algorithm whiz, making this book more suitable for newbies.
+- Thank you to my mentor at the company, Dr. Xi Li, for encouraging me to "get moving" during one of our conversations, which strengthened my resolve to write this book.
+- I would like to thank my girlfriend Bubbles for being the first reader of this book, and for making many valuable suggestions from the perspective of an algorithm whiz, making this book more suitable for beginners.
 - Thanks to Tengbao, Qibao, and Feibao for coming up with a creative name for this book that evokes fond memories of writing the first line of code "Hello World!".
 - Thanks to Sutong for designing the beautiful cover and logo for this book and patiently revising it many times under my OCD.
 - Thanks to @squidfunk for writing layout suggestions and for developing the open source documentation theme [Material-for-MkDocs](https://github.com/squidfunk/mkdocs-material/tree/master).
 
-During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. I would like to thank all my teachers and predecessors for their outstanding contributions!
+During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. Thanks for all who came before me for their outstanding contributions!
```

Comment:
"During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. Thanks for all who came before me for their outstanding contributions!" can be changed to ->


"Throughout the writing journey, I delved into numerous textbooks and articles on data structures and algorithms. These resources served as exemplary models, ensuring the accuracy and quality of this book's content. I extend my gratitude to all who preceded me for their invaluable contributions!"